.editorconfig

```diff
@@ -408,3 +408,7 @@ dotnet_diagnostic.IDE0290.severity = suggestion
 # Collection initialization can be simplified
 dotnet_diagnostic.IDE0300.severity = suggestion
 dotnet_diagnostic.IDE0301.severity = suggestion
+
+# https://github.com/xunit/xunit/issues/2780
+[*.cs]
```

Comment:
```suggestion
```

This should be able to be folded into the previous pattern match.

Reply:
Revert this